.github/workflows/release-build-publish.yml

```
@@ -46,9 +47,5 @@ jobs:
       - name: Check Contract Syntax
         run: ./gradlew p8eClean p8eCheck --info
 
-        # in order to bootstrap in github actions - we must have an object-store and query node that we can hit
       - name: Bootstrap Test
-        run: CHAIN_ID=$TEST_CHAIN_ID ENCRYPTION_PRIVATE_KEY=$TEST_ENCRYPTION_PRIVATE_KEY SIGNING_PRIVATE_KEY=$TEST_SIGNING_PRIVATE_KEY ./gradlew p8eBootstrap --info
-
-#      - name: Bootstrap Prod
-#        run: CHAIN_ID=$PROD_CHAIN_ID ENCRYPTION_PRIVATE_KEY=$PROD_ENCRYPTION_PRIVATE_KEY SIGNING_PRIVATE_KEY=$PROD_SIGNING_PRIVATE_KEY ./gradlew p8eBootstrap --info
\ No newline at end of file
+        run: CHAIN_ID=$CHAIN_ID ENCRYPTION_PRIVATE_KEY=$ENCRYPTION_PRIVATE_KEY SIGNING_PRIVATE_KEY=$ENCRYPTION_PRIVATE_KEY IS_TEST=${{ github.event.inputs.publishing_environment == 'testnet' }} ./gradlew p8eBootstrap --info
```

If you do define `IS_TEST`, may want to update this to
```suggestion
        run: CHAIN_ID=$CHAIN_ID ENCRYPTION_PRIVATE_KEY=$ENCRYPTION_PRIVATE_KEY SIGNING_PRIVATE_KEY=$ENCRYPTION_PRIVATE_KEY IS_TEST=$IS_TEST ./gradlew p8eBootstrap --info
```